getting-started/your-first-feature/1_template.md

```diff
@@ -0,0 +1,44 @@
+As mentioned in the [application architecture](https://docs.daml.com/getting-started/app-architecture.html) section, the DAML code defines the data and workflow of the application. The workflow aspect refers to the interactions between parties that are permitted by the system. In the context of a messaging feature, these are essentially the authorization and privacy concerns listed above.
+
+For the authorization part, we take the following approach: a user Bob can message another user Alice when Alice starts following Bob back. When Alice starts following Bob back, she gives permission or authority to Bob to send her a message.
```

Comment:
'permission or authority'? maybe 'permission and authority', or just permission, or just authority.

Reply:
Wrote "permission and authority"